.wp-env.json

```
@@ -0,0 +1,20 @@
+{
+  "env": {
+    "development": {
+      "plugins": [
+        ".",
+        "https://downloads.wordpress.org/plugin/gutenberg.12.2.0.zip",
+        "https://downloads.wordpress.org/plugin/pwa.0.6.0.zip"
+      ]
+    },
+    "tests": {
+      "plugins": [
+        ".",
+        "https://downloads.wordpress.org/plugin/gutenberg.12.2.0.zip"
+      ],
```

Shouldn't the latest versions be used?

```suggestion
        "https://downloads.wordpress.org/plugin/gutenberg.zip",
        "https://downloads.wordpress.org/plugin/pwa.zip"
      ]
    },
    "tests": {
      "plugins": [
        ".",
        "https://downloads.wordpress.org/plugin/gutenberg.zip"
      ],
```

---

Yes, I think it's best to use the latest versions 👍